seahorn/include/proof_allocators.h

```diff
@@ -8,26 +8,30 @@
 #include <nondet.h>
 #include <stdlib.h>
 
-/**
- *
- */
 #define OBJECT_BITS 8
 #define MAX_MALLOC (SIZE_MAX >> (OBJECT_BITS + 1))
 
 #define MEM_BLOCK 4096
 
 void *realloc(void *ptr, size_t new_size);
 
+
 /**
- *
+ * Deterministically allocates [size] bytes and returns a pointer;
+ * memory allocated will be marked as containing non-det content using memhavoc
  */
-void *bounded_calloc(size_t num, size_t size);
+void *bounded_malloc(size_t size);
 
 /**
- *
+ * XXX: no implementation yet
```

Comment:
what is this comment for?

Reply:
do you recall the purpose of this function? I can't find the implementation or a usage for it. maybe just remove?

Reply:
`pure` == malloc without `havoc`. If it is not used, we do not need a prototype. I think this is all got improved with `sea_allocators` that provide all kind of specialized allocator functions.

Reply:
yeah, that's what I initially think this could be for, but can't find a usage; I will remove `bounded_malloc_pure` for now and merge.